### PR TITLE
Installation tips from my experience getting EXLA to compile

### PR DESCRIPTION
**Pull request by TheFirstAvenger**

### CODE REVIEW

File: exla/README.md [hunk 1]
```diff
@@ -35,6 +35,17 @@ If running on Windows, you will also need:
...
 The first compilation will take a long time, as it needs to compile parts of Tensorflow + XLA. Subsequent commands should be much faster.
 
+#### Installation troubleshooting
+
+  * Bazel
+    * Use `bazel --version` to check your Bazel version, make sure you are using v3.1-v3.7.
+    * It can be installed with `asdf`:
+      * asdf plugin-add bazel
+      * asdf install bazel 3.1.0
+      * asdf global bazel 3.1.0
```
Comment (seanmor5) [inline exla/README.md:45]: I would also add that most of the binaries are available on Github: https://github.com/bazelbuild/bazel/releases

**Review by seanmor5 — COMMENTED** [review]
Thanks, I think this section can be helpful while we work out a more streamlined installation process moving forward! Just a couple of nitpicks, nothing crazy

File: exla/README.md (hunk 1 above)
Comment (TheFirstAvenger) [reply exla/README.md:45]: Done!

**Comment by TheFirstAvenger** [comment]
@seanmor5 Thanks for the review, updated as suggested!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**